label_studio/data_import/models.py

```diff
@@ -18,10 +19,16 @@
 logger = logging.getLogger(__name__)
 
 
+def upload_name_generator(instance, filename):
+    return os.path.join(
+        settings.UPLOAD_DIR, str(instance.project_id) + '-' + str(uuid.uuid4())[0:8] + '-' + filename
```

Comment:
can we do here `{str(instance.project)}/{str(uuid.uuid4())[0:8] + '-' + filename}` ?